### ISSUE TITLE
Screenshot issue

### ISSUE THREAD

**Issue by Kupo91** [libretro/easyrpg-libretro#43]
Thats my issue:

![a4b579d94f5be852cda13a349fcb43c085c83eb9](https://user-images.githubusercontent.com/41426906/62304702-cd9ce000-b47e-11e9-808c-bda33df40e79.png)

Im running retroarch in libreelec. And since the developer updated his build my screenshots in easyrpg look like this. I think its because of his build because I tested it with the same retroarch version on windows and dont have this issue there.
I already reported this issue in his forum but I don't expect any help from him.

Maybe you know what could cause this issue?

**Comment by Ghabry** [comment]
Wow this looks strange.
Can you check first if the libretro core is already on 0.6.1?
One way to figure this out is checking whether "Full Heal" is available in the debug scene. 

Or look in the easyrpg_log.txt or boot a folder that has no game in it and select "About".

**Comment by Kupo91** [comment]
Im currently not at home I will check that tomorrow. However Im actually sure that its 0.6.1 because the zip file which I downloaded from buildbot is still on my laptop and its from 26th of july.

I first thought that it is because of this new version but then I tried it with the old core and still had this issue. So both cores showed this behavior.
And also the controls are acting different. I usually open retroarch menu by Start+Select. However if I do so it acts as I press x or a or whatever is mapped to action button. I tried to take a screenshot of the debug menu but I couldnt because when I pressed Start+Select it went 1 action forward.
I already had this issue months ago but at some point it got fixed. Now its back.

Maybe I just should wait for the next retroarch stable and then try again. I read that it should be released next week.

**Comment by Ghabry** [comment]
Maybe they are registered as keyboard keys. Try selecting "Retropad only" in the core options.

**Comment by Kupo91** [comment]
I just checked the log file and I use 0.6.1. However 0.6.0 shows the same issue. 

Im pretty sure that I didnt have this issue before I updated this Libreelec build. He once said that cores from buildbot might not work properly because he has to compile them for this build by himself. However this is months ago and I never had issues with easyrpg. He meant that the cores still work on LE 8.2 base and not on LE 9. But that was in january.

Selecting "Retropad only" unfortunatelly didnt bring the solution.

I just found my old bug report which was also about these both topics. This button issue got fixed by something else in the past. Look my 2nd comment here https://github.com/libretro/easyrpg-libretro/issues/19
